be/src/pipeline/pipeline_fragment_context.cpp

```diff
@@ -707,6 +707,16 @@ Status PipelineFragmentContext::submit() {
     }
 }
 
+void PipelineFragmentContext::close_sink() {
+    if (_sink) {
+        if (_prepared) {
```

Comment:
maybe double close sink, the line:726 close one time? rethink the logic ?

Reply:
_sink->close method will check _closed flag, so no worry